index.html

```diff
@@ -3,47 +3,256 @@
+	{"state":"Delaware","employment":458.6,"region":"South"},
+	{"state":"District of Columbia","employment":773.8,"region":"South"},
+	{"state":"Florida","employment":8252.8,"region":"South"},
+	{"state":"Georgia","employment":4360.7,"region":"South"},
+	{"state":"Hawaii","employment":651.4,"region":"West"},
+	{"state":"Idaho","employment":691.7,"region":"West"},
+	{"state":"Illinois","employment":6006.7,"region":"Midwest"},
+	{"state":"Indiana","employment":3062.1,"region":"Midwest"},
+	{"state":"Iowa","employment":1576.5,"region":"Midwest"},
+	{"state":"Kansas","employment":1399.2,"region":"Midwest"},
+	{"state":"Kentucky","employment":1908.9,"region":"South"},
+	{"state":"Louisiana","employment":1977.1,"region":"South"},
+	{"state":"Maine","employment":614.6,"region":"Northeast"},
+	{"state":"Maryland","employment":2704.6,"region":"South"},
+	{"state":"Massachusetts","employment":3537.1,"region":"Northeast"},
+	{"state":"Michigan","employment":4324.9,"region":"Midwest"},
+	{"state":"Minnesota","employment":2876.2,"region":"Midwest"},
+	{"state":"Mississippi","employment":1147.1,"region":"South"},
+	{"state":"Missouri","employment":2801.2,"region":"Midwest"},
+	{"state":"Montana","employment":464.6,"region":"West"},
+	{"state":"Nebraska","employment":1017.3,"region":"Midwest"},
+	{"state":"Nevada","employment":1282.7,"region":"West"},
+	{"state":"New Hampshire","employment":664.9,"region":"Northeast"},
+	{"state":"New Jersey","employment":4073.9,"region":"Northeast"},
+	{"state":"New Mexico","employment":827.6,"region":"West"},
+	{"state":"New York","employment":9332.9,"region":"Northeast"},
+	{"state":"North Carolina","employment":4310.5,"region":"South"},
+	{"state":"North Dakota","employment":440.0,"region":"Midwest"},
+	{"state":"Ohio","employment":5491.4,"region":"Midwest"},
+	{"state":"Oklahoma","employment":1666.0,"region":"South"},
+	{"state":"Oregon","employment":1823.2,"region":"West"},
+	{"state":"Pennsylvania","employment":5894.6,"region":"Northeast"},
+	{"state":"Rhode Island","employment":490.6,"region":"Northeast"},
+	{"state":"South Carolina","employment":2039.6,"region":"South"},
+	{"state":"South Dakota","employment":432.3,"region":"Midwest"},
+	{"state":"Tennessee","employment":2951.7,"region":"South"},
+	{"state":"Texas","employment":11961.6,"region":"South"},
+	{"state":"Utah","employment":1411.7,"region":"West"},
+	{"state":"Vermont","employment":316.5,"region":"Northeast"},
+	{"state":"Virginia","employment":3919.1,"region":"South"},
+	{"state":"Washington","employment":3230.5,"region":"West"},
+	{"state":"West Virginia","employment":761.8,"region":"South"},
+	{"state":"Wisconsin","employment":2936.2,"region":"Midwest"},
+	{"state":"Wyoming","employment":284.4,"region":"West"}
+  ];
+
+  var MAX_EMPLOYMENT = 16322;
+
+  var options = {
+    sorted: false,
+    filtered: false,
+    page: 0
+  };
+    
+    
+    var tbody = d3.select("tbody");
+
+  function updateTable(){
+    var rows = "";
```

Comment:
To create a variable without assigning it a value, we can just do: `var rows;`. But we could also just put the `var` statement where we first use `rows`, on line 147.